debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoUtil.java

```diff
@@ -80,7 +80,7 @@ public static String replicaSetUsedIn(String addresses) {
     /**
      * Perform the given operation on each of the database names.
      *
-     * @param client the MongoDB client; may not be null
+     * @param client    the MongoDB client; may not be null
```

Comment:
There are quite a number of java doc indentation changes here that create a lot of unnecessary noise.  Can you please revert these types of changes as they're superfluous?